packages/web/src/components/nav/desktop/NavPopupMenu.tsx

```diff
@@ -62,7 +66,10 @@ const NavPopupMenu = () => {
     isChatEnabled
       ? {
           text: messagesItemText,
-          onClick: () => navigate(CHATS_PAGE),
+          onClick: () => {
+            dispatch(make(Name.CHAT_ENTRY_POINT, { source: 'navmenu' }))
+            navigate(CHATS_PAGE)
```

Comment:
ultra nit, curious for your persp. on. i like the idea of doing "UX" things first, e.g. we shouldn't delay navigation at all costs. in practice i dont think this matters here, but just figured i'd write

Reply:
I usually do the same, this was mistake - updated